src/dtgtk/thumbtable.c

```diff
@@ -1061,22 +1061,31 @@ static gboolean _event_scroll_compressed(gpointer user_data)
     // half shown if scrollbar used)
     int move = table->thumb_size * delta;
 
-    // if the top thumb row is only partially visible, then realign first
-    const int partial_height = table->thumbs_area.y % table->thumb_size;
-    if(partial_height)
-    {
-      if(delta < 0)
-      {
-        move = partial_height;
-      }
-      else
+    // clicky scroll wheels generate integer clicks, scroll by one thumb size:
+    if (fabsf(delta) == 1.f) {
+      // if the top thumb row is only partially visible, then realign first
+      const int partial_height = table->thumbs_area.y % table->thumb_size;
+      if(partial_height)
       {
-        move = table->thumb_size + partial_height;
+        if(delta < 0)
+        {
+          move = partial_height;
+        }
+        else
+        {
+          move = table->thumb_size + partial_height;
+        }
       }
+      _move(table, 0, -move, TRUE);
+    } 
+    // precision touch pads generate float increments, scroll by pixel delta:
+    else {
+#ifdef GDK_WINDOWING_QUARTZ // on macOS deltas need to be scaled
+      delta *= 50; // see dt_gui_get_scroll_deltas
```

Comment:
It appears that Windows scroll events need to be scaled likewise. At least on my Surface Pro 11, which I think has a certified precision touch pad.

Can someone else verify this?

Reply:
I have changed the scaling to be platform-independent. This seems to do the right thing on Windows and macOS.